aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -582,6 +588,11 @@ module aptos_framework::fungible_asset {
         store: Object<T>,
         amount: u64,
     ): FungibleAsset acquires FungibleStore, DispatchFunctionStore {
+        // ** IMPORTANT **
+        //
+        // For any changes in the access control logic here, make sure we duplicate them in
+        // dispatchable_fungible_asset::withdraw as that will be a separate entrypoint.
+        //
         assert!(object::owns(store, signer::address_of(owner)), error::permission_denied(ENOT_STORE_OWNER));
         assert!(store_exists(object::object_address(&store)), error::invalid_argument(ESTORE_IS_FROZEN));
```

Comment:
As I said in the landed PR, `borrow_store_resource` checks for you, please remove similar checks in this PR